docs/docs/v1.0.x/in-repo-addon-plugin.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Creating a plugin as an in-repo-addon
+---
+
+So, you want to prototype a plugin or write one that is so specific to your app that it's not worth making a new npm package for?
```

Comment:
Maybe be a bit more descriptive about use cases, _what_ are causes that makes this so easy to start with, so specific? To draw the line between _this needs to be in-repo_ and _this is useful to the community_